src/main/java/net/fabricmc/mappingio/format/StandardProperties.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2023 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.mappingio.format;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.Nullable;
+
+public final class StandardProperties {
+	private StandardProperties() {
+	}
+
+	public static Set<StandardProperty> values() {
+		return Collections.unmodifiableSet(values);
```

Comment:
Is there a particular reason we should use a Set instead of just using `valuesByName.values()` or `valuesById.values()`?

Reply:
Honestly, I don't remember. Might have been an oversight